### PR TITLE
Menu mobile

### PR DESCRIPTION
**Pull request by OsvaldoBC**
In this pull request, we add the portfolio mobile menu

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,23 +1,51 @@
...
+  box-sizing: border-box;
+  padding: 40px;
+  transition: 250ms height ease-in-out;
+}
+
+.box-menu-1 a {
+  color: white;
+  text-decoration: none;
+  font-size: 32px;
+}
```
Comment (codepantha) [inline css/styles.css]: - [x] Kindly make the font weight of the mobile menu links to be `font-weight: 600` as specified in the Figma design to follow the specifications 👍 
- [x] Also take care to implement the margin between the links so they appear as specified in the design template
## Figma Design
image

## Your Design
image

File: css/styles.css [hunk 2]
```diff
@@ -1,23 +1,51 @@
...
+  background-color: #6070ffe3;
+  margin: 0;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 40px;
+  transition: 250ms height ease-in-out;
+}
```
Comment (codepantha) [inline css/styles.css]: - [x] Please let's follow the design guidelines and implement the blur on the mobile menu as shown in the template 🙏 
**HINT:** Use `backdrop-filter: blur(specify the blur in px)` 😎 
## Figma Design
image

## Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Good commit history

## Required Changes ♻️
_Check the comments under the review._

- [x] Since it's a collaborative project, kindly include your partner's details in the "Authors" section of the README 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌 Professional README ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**